scripts/api/activateVersion.ts

```diff
@@ -0,0 +1,8 @@
+import { createClient } from '../utils/createClient'
+
+export async function activateVersion(service_id: string, versionId: number) {
```

Comment:
nitpick: no need for `async` here

Reply:
This commit refactors it: https://github.com/fingerprintjs/fingerprint-pro-fastly-proxy-integration-example/pull/7/commits/5ec6135dc3a6a23be6413f407fdefad2dde93aac